src/public/stageView.ts

```diff
@@ -0,0 +1,38 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { SdkError } from '../public';
+import { FrameContexts } from './constants';
+
+/**
+ * Namespace to interact with the stage view specific part of the SDK.
+ */
+export namespace stageView {
+  /**
+   * @private
+   * Feature is under development
+   *
+   * Opens a stage view to display a Teams app
+   * @param applicationId The ID of the Teams app
+   * @param context The context required to launch the Teams app
+   * @param callback Callback that will be triggered once the stage view is closed.
+   *                 The callback takes as an argument an SdkError in case something happened (i.e.
+   *                 no permissions to execute the API)
+   */
+  export function openStageView(
```

Comment:
It seems very similar to what dialog/task module is, so maybe we should try to have the API be similar too? `stageView.open` to align with `dialog.open`?